content/post/2022-01-03-mlr3-package-updates-q4-2021/index.Rmd

```diff
@@ -0,0 +1,85 @@
+---
+title: mlr3 package updates - Q4/2021
+author: Patrick Schratz
+date: '2022-01-03'
+slug: mlr3-package-updates-q4-2021
+categories:
+  - R
+tags: []
+authors: []
+---
+
+Due to the high amount of packages in the mlr3 ecosystem, it is hard to keep up with the latest changes across all packages.
+This post tries to tackle this issue by listing all release notes of the packages most recent releases in the last quarter.
```

Comment:
packages'

Reply:
This posts gives you an overview by listing the recent release notes of mlr3 packages from the last quarter.

Reply:
Danke, können den intro Text gerne updaten, der sollte bei jedem Post dann immer nur c/p rein.